framework/scripts/wazuh_server.py

```diff
@@ -196,6 +202,7 @@ async def master_main(args: argparse.Namespace, server_config: ServerConfig, log
     from wazuh.core.cluster import local_server, master
 
     context_tag.set('Master')
+    start_unix_server('Master')
```

Comment:
```suggestion
    tag = 'Master'
    context_tag.set(tag)
    start_unix_server(tag)
```

Reply:
Changed in edcb389a839c82846356d2ca9e981f422df1c442.